src/main/java/Deadline.java

```diff
@@ -0,0 +1,41 @@
+//package duke;
+/**
+ * Deadline <- Task
+ */
+public class Deadline extends Task {
```

Comment:
I like how all the class names in your code and clear, concise and adhered to the Java coding standards and naming conventions.